concent_api/core/models.py

```diff
@@ -485,3 +493,478 @@ def clean(self):
             raise ValidationError({
                 'pending_response': 'PaymentInfo should be related with Pending Response'
             })
+
+
+class AbstractMessageModel(Model):
+
+    sig = BinaryField()
+    timestamp = IntegerField()
+    encrypted = BooleanField(default = False)
+
+    class Meta:
+        abstract = True
+
+
+class StoredComputeTaskDef(AbstractMessageModel):
+
+    # Defines items list from ComputeTaskDef
+    ITEMS = [
+        'task_id',
+        'subtask_id',
+        'deadline',
+        'src_code',
+        'extra_data',
+        'short_description',
+        'working_directory',
+        'performance',
+        'docker_images',
+    ]
+
+    task_id = CharField(max_length=MESSAGE_TASK_ID_MAX_LENGTH)
+    subtask_id = CharField(max_length=MESSAGE_TASK_ID_MAX_LENGTH)
+    deadline = DateTimeField(blank = True, null = True)
+    src_code = TextField()
+    extra_data = JSONField()
+    short_description = TextField()
+    working_directory = TextField()
+    performance = DecimalField(decimal_places=2, max_digits=3)
+    docker_images = JSONField()
+
+    if not len(ITEMS) == len(message.tasks.ComputeTaskDef().ITEMS.keys()):
+        raise ValidationError({
+            'ComputeTaskDef': 'StoredComputeTaskDef model has not same fields as golem_messages.message.tasks.ComputeTaskDef()'
+        })
+
+    for model_item, golem_message_item in zip(ITEMS, message.tasks.ComputeTaskDef().ITEMS.keys()):
+        if not model_item == golem_message_item:
+            raise ValidationError({
+                'ComputeTaskDef': 'StoredComputeTaskDef model has not same fields as golem_messages.message.tasks.ComputeTaskDef()'
```

Comment:
I think it would be better to replace both of those checks with one that creates sets of field names and compares them:

``` python
assert set(ITEMS) == set(message.tasks.ComputeTaskDef().ITEMS):
```